Lesson 4/vladyslav.volkov/nginx_run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+if nginx -v >/dev/null 2>&1; then
+
+	nginx_ver=$(nginx -v 2>&1)
+	echo "Removing $nginx_ver"
+	apt-get purge nginx -y
+else
+
+	echo "Nginx is not installed"
+
+fi
+
+apt update
+apt install curl gnupg2 ca-certificates lsb-release -y
+echo "deb http://nginx.org/packages/ubuntu `lsb_release -cs` nginx" \
+    | tee /etc/apt/sources.list.d/nginx.list
+curl -fsSL https://nginx.org/keys/nginx_signing.key | apt-key add -
+apt install nginx=1.14.2-1~xenial
```

Comment:
Reading state information... Done
E: Version '1.14.2-1~xenial' for 'nginx' was not found
mkdir: cannot create directory '/etc/nginx/sites-available': No such file or directory
mkdir: cannot create directory '/etc/nginx/sites-enabled': No such file or directory